README.md

```diff
@@ -17,31 +18,25 @@ In the root directory, run:
 
 ### Normal updating
 
-- To update schedules, run `python update.py`. This runs three steps:
-  - `python scrapers/coursews.py` to get the class schedules.
-  - `python scrapers/catalog.py` to get the class descriptions.
-  - `python scrapers/package.py` to bundle schedules, descriptions, and evaluations into a `json` file.
-- To update evaluations, run `python update_evals.py`.
-- To update the frontend, run `npm build`. Then run `./deploy.sh` or something.
+- To update schedules, `cd scrapers` then run `python update.py`.
+  - This updates the file `public/latest.json`.
+  - It's important for the working directory to be `scrapers`.
+  - On the server, there should be a cron script running this every few hours.
+- To update the frontend, run `npm build`.
+  - Then run `./deploy.sh` or something to deploy it. Need to figure this out.
```

Comment:
@GenericP3rson how's this?

Reply:
the text is not quite right (it should be `full.json` not `latest.json`) but it will be once #8 gets merged oops

Reply:
Looks good! The file's running now!